lib/node.js

```diff
@@ -181,28 +181,21 @@ class Node {
     }
 
     switch (data.event) {
-      case 'autorefresh':
-        return this.redis
-          .hgetall('cluster:autorefresh')
-          .then(autoRefreshes => {
-            for (const index of Object.keys(autoRefreshes)) {
-              const autoRefresh = autoRefreshes[index];
-              this.kuzzle.services.list.storageEngine.setAutoRefresh(new Request({
-                index,
-                controller: 'index',
-                action: 'setAutoRefresh',
-                body: {autoRefresh}
-              }));
-            }
-          });
       case 'indexCache:add':
-        this.kuzzle.indexCache.add(data.index, data.collection, false);
+        this.kuzzle.indexCache.add({
+          index: data.index,
+          collection: data.collection,
+          scope: data.scope,
+          notify: false,
+        });
         break;
       case 'indexCache:remove':
-        this.kuzzle.indexCache.remove(data.index, data.collection, false);
-        break;
-      case 'indexCache:reset':
-        this.kuzzle.indexCache.reset();
+        this.kuzzle.indexCache.remove({
+          index: data.index,
+          collection: data.collection,
+          scope: data.scope,
+          notify: false,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this trailing comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ATrailingComma)